projects/examples/src/components/action-menu/action-menu.example.component.ts

```diff
@@ -44,7 +55,8 @@ interface HandlerData {
             <button (click)="toggleDropdownDisable()" class="btn btn-primary">
                 {{ isDropdownDisabled ? 'Enable dropdown' : 'Disable dropdown' }}
             </button>
-            <br />
+            <br /><br />
+            <p>Press 'mod+.' on the keyboard to open quick search and search for contextual actions</p>
```

Comment:
I'm about to change this at that other place. Sorry for providing bad source to copy from :( 
'mod' is internal. Better say Press 'command+.' on Mac OS /'ctrl+.' on PC/ ...